pkg/app/instances/docker.go

```diff
@@ -117,16 +119,56 @@ func (m *DockerInstanceManager) DeleteHost(zone string, _ accounts.User, host st
 		return nil, fmt.Errorf("Failed to remove docker container: %w", err)
 	}
 	return &apiv1.Operation{
-		Name: host,
-		Done: true,
+		Name: "deletehost_" + host,
+		Done: false,
 	}, nil
 }
 
-func (m *DockerInstanceManager) WaitOperation(zone string, _ accounts.User, _ string) (any, error) {
+func (m *DockerInstanceManager) WaitCreateHostOperation(host string) (any, error) {
+	ctx := context.TODO()
+	for {
```

Comment:
let's not have an infinite loop here. let's have a fixed number of retries, if the loop ends we return `create failed`.

Reply:
Instead introducing fixed number of retries, set timeout for the function.

Reply:
better, thanks!